src/Microsoft.Health.Fhir.Shared.Core/Extensions/ReindexJobRecordExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Operations.Reindex.Models;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Extensions
+{
+    public static class ReindexJobRecordExtensions
+    {
+        public static ResourceElement ToParametersResourceElement(this ReindexJobWrapper record)
+        {
+            EnsureArg.IsNotNull(record, nameof(record));
+
+            var parametersResource = new Parameters();
```

Comment:
Not needed yet, but I challenge you to create a Parameters object like below, but by using ElementNode :)